packages/go_router/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Updates minimum supported SDK version to Flutter 3.16/Dart 3.2.
 - Fixes memory leaks.
+- GoRouter.optionURLReflectsImperativeAPIs now works correctly with new imperative APIs
```

Comment:
Nit: This description sounds like you changed the "optionURLReflectsImperativeAPIs" flag. You can update the description like "Fixes `restoreRouteInformation` issue when GoRouter.optionURLReflectsImperativeAPIs is true and the last match is ShellRouteMatch"

Reply:
done